platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -421,4 +428,8 @@ protected BundleContext getContext() {
   void setServiceTracker(ServiceTracker serviceTracker) {
     this.serviceTracker = (ServiceTracker<Object, Object>) serviceTracker;
   }
+
+  private static String computeLocation(Bundle bundle) {
```

Comment:
Bundle location caching is duplicated in the application service code but this code is deprecated for removal.